src/utils/company-type.ts

```diff
@@ -0,0 +1,92 @@
+import { CorpTypeCd } from '@bcrs-shared-components/enums'
+
+/** Whether the entity is a Benefit Company. */
+export const isEntityBenefitCompany = (legalType: CorpTypeCd): boolean => {
```

Comment:
Hey so we have a legal type comparison function in the business store already can we just use that instead of all these individual ones please? Keep in mind there are many more of these that'll be pulled over from COLIN